projects/pgai/tests/vectorizer/cli/test_vectorizer_document.py

```diff
@@ -18,6 +18,7 @@
     "sample_txt.txt",
     "basic-v3plus2.epub",
     "test.md",
+    "NVDA.pdf",
```

Comment:
I am not sure we can just add this file under our main license?. Sure it's a publicly readable google doc with public data in it and unlikely someone complains but I didn't create this so I'm a bit careful about that 😅

Reply:
Yeah don't worry, this pr won't be merged with that spreadsheet but instead will create a custom one.
Thanks for pointing that out.

Reply:
Replaced by `lego_sets.pdf`, a PDF based from a free dataset from https://mavenanalytics.io/data-playground.
I imported the CSV in Google Spreadsheets, added an image (to test OCR on/off) and a title, and limited the dataset to few Star Wars lego sets to reduce the size.